index.html

```diff
@@ -55,7 +55,7 @@ <h1>I'm Tesfaye <br>Glad to see you!</h1>
         </ul>
       </section>
     </main>
-
+<div class="card-container">
```

Comment:
- [ ] Kindly make sure your card_works are well positioned as the ones on the template in Figma.
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193204384-0e035888-7a55-433c-be5e-f9687721cc17.png)
 Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193204519-c512e899-240c-410b-816e-2188c45a33ab.png)